qcodes/instrument_drivers/AlazarTech/constants.py

```diff
@@ -519,3 +519,83 @@ class RecordAverageOption(IntEnum):
 LED_OFF = 0
 LED_ON = 1
 
+
+class Parameters(IntEnum):
```

Comment:
with all the other enum classes defined before this PR, i was trying to follow the convention that the name of the class represents what one of the elements is, hence it's single (not plural). So, i'd suggest to rename this class to `Parameter` (not a good name in the qcodes realm but whatever), and the other new enums to "not plural".